client/src/components/ReviewList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Fragment } from 'react';
+import ReviewListEntry from './ReviewListEntry.jsx';
+
+class ReviewList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      
+    };
+  }
+
+  componentDidMount() {
+  }
+
+  // HTTP Request Handlers
+  // Set State
+  // Event Handlers
+
+  render() {
+    return (
+      <Fragment>
+        <div className="nav-sort">
+          <div>sort on</div>
+          <button>newest</button>
+          <button>helpful</button>
+          <button>relevant</button>
+        </div>
+        {this.props.reviews.map((review) => 
+          <ReviewListEntry key={review.id} review={review}/>
```

Comment:
I think this is a smart use of the review.id to silence the key prop warning. Just a random FYI in case you don't already know this, the key prop is NOT accessible from within the child component. So if you needed to access the review.id inside of ReviewListEntry (which you don't since you're passing the whole review object), you'd have to pass it as a different prop: `key={review.id} id={review.id}`.

Reply:
Yup! Just using the id to silence the key warning. I'll be accessing the id through props.review.id instead of props.id